gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
Hi Tony, [any web test](https://github.com/the-t-in-rtf/universal/blob/GPII-2307/tests/web/html/all-tests.html#L13-L37) listed in `tests/web/html/all-tests.html` will fail with the similar error messages that can be found in the screenshot below. This screenshot is the result of running the very first web test listed in the `all-tests.html` file in a browser:

![screen shot 2017-05-23 at 3 23 43 pm](https://cloud.githubusercontent.com/assets/572877/26372426/f1148d14-3fcb-11e7-8891-96189c8fae45.jpeg)

As mentioned earlier of my use case, my document root directory defined in MAMP Apache is a couple of levels higher than the directory where `universal` repo is cloned. When I use MAMP served localhost to open a test file in a browser, the use of absolute paths for `/testem.js` and `/coverage/client/coverageSender.js` would not locate them at that dir level, which causes these errors to occur.